src/App.js

```diff
@@ -69,33 +83,52 @@ const App = ({ setClient }) => {
   return <RootNavigator initialRoute={initialRoute} setClient={setClient} />
 }
 
-const Nav = ({ client, setClient }) => {
+// eslint-disable-next-line react/display-name
```

Comment:
You shouldn't deactivated this lint error as it impacts debug readability.

You can do instead: `InnerNave.displayName = 'InnerNav'`

But I don't understand why this is needed now. It is supposed to be needed when you wrap components in methods like `forwardRef` but there seems to be nothing like this here.

Reply:
I removed it, it was indeed an artifact from my memoization solution for stopping rerenders